tweetapi/views.py

```diff
@@ -9,28 +9,33 @@
 from .serializer import ProfileSerilizer,TweetSerializer,CommentSerializer,RetweetSerializer,LikeSerializer,UserSerializer
 # Create your views here.
 
-class UserView(generics.ListCreateAPIView):
-    queryset = User.objects.all()
+class UserView(generics.ListAPIView):
+    queryset = User.objects.prefetch_related("user_profile")
```

Comment:
that should be select related

Reply:
why is it a list view?

Reply:
ITS JUST FOR MY PERSONAL USE SIR, JUST WANTED TO SEE ALL THE USERS I CREATED WHILE I WAS TESTING..

Reply:
its is good practice just not used correctly